packages/react/src/components/chart/BarSeries.tsx

```diff
@@ -1,4 +1,4 @@
-import {range, scaleBand} from 'd3';
+import * as d3 from 'd3';
```

Comment:
Not sure about the impact on the distributed bundle size, ideally we would `import {range, scaleBand} from 'd3-array'` or something alike.

Reply:
i'll try again then 👍

Reply:
i think I succeeded
<img src="https://media4.giphy.com/media/3o6nV28eyNyk7CGk24/giphy.gif"/>